xdsl/dialects/arm/__init__.py

```diff
@@ -8,7 +8,7 @@
 from xdsl.dialects.builtin import ModuleOp
 from xdsl.ir import Dialect
 
-from .ops import ARMOperation, DSMovOp, DSSMulOp, GetRegisterOp
+from .ops import ARMOperation, DSMovOp, DSSMulOp, GetRegisterOp, LabelOp
```

Comment:
Feels like this could be its own PR

Reply:
this is actually its own PR now (https://github.com/xdslproject/xdsl/pull/3749), and I think all of your other comments are resolved in that one. apologies for the confusion, I will clearly mark WIP PRs from now on